index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <title>Microverse Portfolio project</title>
+  </head>
+  <body>
+    <header>
+      <div class="Logo">
+        <h4><a href="#"></a>Solomon Afutu</h4>
```

Comment:
- [ ] As mentioned by previous reviewer, the logo should be a link tad `a` navigating to the home page i.e. `index.html`  This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. I suggest you put logo nme inside of `a` tag, right now its written outside the `a` tag. also use `href ` attribute to navigate to the home page.
``` sh
<h4><a href="index.html">Solomon Afutu</a></h4>
```